pkgs/development/interpreters/racket/default.nix

```diff
@@ -60,6 +60,8 @@ stdenv.mkDerivation rec {
     for p in $(ls $out/bin/) ; do
       wrapProgram $out/bin/$p --set LD_LIBRARY_PATH "${LD_LIBRARY_PATH}";
     done
+    export PATH="$out/bin:$PATH";
```

Comment:
@henrytill I re-ran with just `"$out/bin/raco" setup`, and it seems to work. I'll force-push when it finishes, if it works until the end.